src/tools/tidy/src/ui_tests.rs

```diff
@@ -7,7 +7,7 @@ use std::path::Path;
 
 const ENTRY_LIMIT: usize = 1000;
 // FIXME: The following limits should be reduced eventually.
-const ROOT_ENTRY_LIMIT: usize = 948;
+const ROOT_ENTRY_LIMIT: usize = 949;
```

Comment:
Can you put the new test in a subdirectory rather than raising the limit? Maybe `compiletest-self-test` or something like that? We could move src/test/ui/ui-testing-optout.rs there at the same time.

Reply:
Done!